INSTALL.md

```diff
@@ -76,13 +74,7 @@ When cross compiling for a different platform, the targeted hardware doesn't nee
 
 #### Building with Clang transplier option
 
-
-Hard dependency is clang-17 exactly. So far clang based transpiler does not have compatibility layer to support differences of C++ API in clang tooling in newer versions. How to install it please refer to the original 
-[clang occa-transpiler](https://github.com/libocca/occa-transpiler/blob/main/README.md)

```

Comment:
Please keep the link to occa-trasnpiler README here as reminder to the user to install it.

Reply:
Fix this in the latest commit.